src/pushi/base/smtp.py

```
@@ -0,0 +1,438 @@
+        smtp://mail.example.com
+        smtp://user:pass@mail.example.com:587?sender=noreply@example.com
+        smtps://user:pass@mail.example.com:587?sender=noreply@example.com
+
+    :see: https://en.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol
+    """
+
+    def __init__(self, owner):
+        handler.Handler.__init__(self, owner, name="smtp")
+        self.subs = {}
+
+    def send(self, app_id, event, json_d, invalid={}):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)